src/emu/emuopts.cpp

```diff
@@ -209,7 +209,7 @@ const options_entry emu_options::s_option_entries[] =
 	{ OPTION_AUTOBOOT_SCRIPT ";script",                  nullptr,     OPTION_STRING,     "lua script to execute after machine boot" },
 	{ OPTION_CONSOLE,                                    "0",         OPTION_BOOLEAN,    "enable emulator LUA console" },
 	{ OPTION_PLUGINS,                                    "1",         OPTION_BOOLEAN,    "enable LUA plugin support" },
-	{ OPTION_PLUGIN,                                     nullptr,     OPTION_STRING,     "list of plugins to enable" },
+	{ OPTION_PLUGIN,                                     "portname",  OPTION_STRING,     "list of plugins to enable" },
```

Comment:
Don't do this.  Just change portname/plugin.json "start" to "true".

Reply:
ahh, didn't know there was a better way :-)